src/alire/alire-config-edit.adb

```diff
@@ -17,6 +17,10 @@ package body Alire.Config.Edit is
    type String_Access is access String;
    Config_Path : String_Access;
 
+   --  Separately loaded configs to be able to ascertain where a config value
+   --  is coming from.
+   Level_DB : array (Level) of CLIC.Config.Instance;
```

Comment:
So there are 3 DBs now [this one](https://github.com/alire-project/alire/blob/5c17abb0dc6ed48158ebd51750761317063b5ccd/src/alire/alire-config.ads#L8) and the two here? It seems overkill.

There's an origin field for every config value, we might be able to use that. Or we remove the common DB and only use the Level_DB with a getter function that will first look in local then global.

Reply:
I see, I had not seen the origin field. Will try to use that.